### DISCUSSION TITLE
Where can I report a typo on the support pages?

### DISCUSSION THREAD

**Discussion by tripleee** [community/community#17686] [Other Feature Feedback, Questions, & Ideas] [answered]
https://support.github.com/contact/report-abuse?category=report-abuse&report=tripleee&report_id=68517177&report_type=user oddly spells "spam" in all upper case. The word is not an acronym, and should not be capitalized.

(Update: The form has several entries which are collapsed; you have to click the "I want to report abusive content or behavior" item to expose the first of the quoted sentences below, and then select the item I quote below that to see the final one.)

https://github.com/github/ generally uses the correct spelling, but I was unable to find this string in a place where I could submit a PR.

I feel the wording of these headings could perhaps also be improved.

> I want to report SPAM, a user that is disrupting me or my organization's experience on GitHub, or a user who is using my personal information without my permission

> A user is disrupting me or my organization's experience and productivity by posting SPAM off-topic or other types of disruptive content in projects they do not own.

The first bullet point might make more sense as e.g.

_A user is posting spam or rude or vandalizing content, or is using my personal information without my permission, or is otherwise disrupting me or other visitors to Github_

and the second is definitely missing a comma.

> A user's profile or repository is SPAM.

This bullet point should use "spam" in lower case.

**Comment by ShobanChiddarth** [comment]
If I click your link, I don't even see the word SPAM anywhere.
![image](https://user-images.githubusercontent.com/68191453/171910520-fc32cdfc-d9d7-4379-92ef-7e57c428531a.png)
This issue is outdated. Please update it.

**Reply by tripleee** [reply]
You have to click open the "I want to report abusive content or behavior" item. Thanks for prodding; I updated the question with these details.

**Reply by ShobanChiddarth** [reply]
Yes, I can see that. Just report it by contacting them.

**Reply by tripleee** [reply]
At the peril of restating the obvious, my actual question is _"Where can I report a typo on the support pages?"_ Can you answer that?

**Reply by ShobanChiddarth** [reply]
https://github.com/github-community/community/discussions

**Reply by tripleee** [reply]
I'm sorry, how is that helpful? That's where you are responding to this.

**Reply by ShobanChiddarth** [reply]
You can report by creating a new discussion from my link

**Reply by tripleee** [reply]
Yes, but why should I create a duplicate? The discussion you are participating in was created exactly by doing that.

<img width="1141" alt="Screenshot 2022-06-15 at 11 56 43" src="https://user-images.githubusercontent.com/2160915/173787509-28c0ac46-7f3b-4424-99d1-6496f1be15bb.png">

**Reply by ShobanChiddarth** [reply]
Then bye

**Comment by iBug** [comment, marked as answer]
While not the most appropriate method, submitting a ticket is the quickest way to get things fixed, particularly this kind of simple issue. The first level of support personnel will sort your ticket out for the right team and not much burden will be on the support system.

**Reply by tripleee** [reply]
Thanks, you mean here? https://support.github.com/tickets/personal

**Reply by tripleee** [reply]
Actually many of the options there are unhelpful, irrelevant, or simply route me back here. I submitted a ticket via https://support.github.com/contact/account now.

**Reply by tripleee** [reply]
Finally, I got a response.

> Thank you for sending this feedback. We have shared it with the appropriate team.
